diff --git a/node_modules/next/dist/server/app-render/action-handler.js b/node_modules/next/dist/server/app-render/action-handler.js
index 45579f6..7a97926 100644
--- a/node_modules/next/dist/server/app-render/action-handler.js
+++ b/node_modules/next/dist/server/app-render/action-handler.js
@@ -530,9 +530,7 @@
         // ensure that the forwarded host is part of the allowed origins.
         // If it's not, we'll forbid the request.
         if (process.env.NEXT_PRIVATE_ALLOWED_origins) {
-            const origin = request.headers.get("origin");
-            const forwardedHost = request.headers.get("host");
-            if (!origin && forwardedHost !== originalHost) {
+            if (process.env.NODE_ENV === 'development' && process.env.NEXT_PRIVATE_DEBUG_TOOLBAR) {
                 (0, _log.warn)(`An action was blocked because the 'host' header is not matching the server's host. This is a security measure to prevent CSRF attacks.`)
                 return {
                     body: _react.default.createElement(_actionRender.ActionForbidden, {
@@ -541,6 +539,9 @@
                     }),
                     isNotFound: true
                 };
+            } else {
+                const origin = request.headers.get("origin");
+                const forwardedHost = request.headers.get("host");
+            if (!origin && forwardedHost !== originalHost) {
                 (0, _log.warn)(`An action was blocked because the 'host' header is not matching the server's host. This is a security measure to prevent CSRF attacks.`)
                 return {
                     body: _react.default.createElement(_actionRender.ActionForbidden, {
@@ -550,6 +551,7 @@
                     isNotFound: true
                 };
             }
+          }
         }
     }
     let boundServerAction;

