--- a/node_modules/next/dist/server/web/sandbox/base-sandbox.js
+++ b/node_modules/next/dist/server/web/sandbox/base-sandbox.js
@@ -191,9 +191,7 @@
         // that the code is being run in a sandbox.
         this.context.process = new Proxy(this.context.process, {
             get: (target, prop, receiver)=>{
-                if (prop === "env") {
-                    return new Proxy(target.env, {
-                        get: (target, prop, receiver)=>{
-                            if (prop === "__NEXT_IS_SANDBOX") {
-                                return "true";
-                            }
-                            // TODO: remove this when we can move to `next-app-loader`
-                            // and `next-edge-app-loader` which will not have this issue.
-                            if (String(prop).startsWith("NEXT_")) {
-                                return undefined;
-                            }
-                            return Reflect.get(target, prop, receiver);
-                        },
-                        has: (target, prop)=>{
-                            if (prop === "__NEXT_IS_SANDBOX") {
-                                return true;
-                            }
-                            if (String(prop).startsWith("NEXT_")) {
-                                return false;
-                            }
-                            return Reflect.has(target, prop);
-                        }
-                    });
-                }
-                return Reflect.get(target, prop, receiver);
+                return Reflect.get(target, prop, receiver)
             }
         });
     }