diff --git a/node_modules/next/dist/server/web/sandbox/sandbox.js b/node_modules/next/dist/server/web/sandbox/sandbox.js
index 45b5974..5f53298 100644
--- a/node_modules/next/dist/server/web/sandbox/sandbox.js
+++ b/node_modules/next/dist/server/web/sandbox/sandbox.js
@@ -299,9 +299,9 @@
     }
     const { run, runWith, dispose, edgeFunctions, wasm, ...relative } = await (0, _edgeBundler).createEdgeFunction({
         ...options,
-        useCache: true,
+        useCache: false,
         // Bundling development is not necessary.
-        development: false
+        development: true
     });
     const webRequest = toWebRequest(request);
     const edgeFunction = {

